app/setup.py

```diff
@@ -225,6 +237,8 @@ def after_request(response):  # pylint: disable=unused-variable
         # We're using the stringified version of the Flask session to get a rough
         # length for the cookie. The real length won't be known yet as Flask
         # serializes and adds the cookie header after this method is called.
+        if flask_request.method == "OPTIONS":
+            contextvars.clear_contextvars()
```

Comment:
Same as above, why is this now needed?. Is there a link to the doc that explains this?.

Reply:
Here is the documentation highlighting the [part](https://www.structlog.org/en/stable/contextvars.html#:~:text=Call%20structlog.contextvars.clear_contextvars()%20at%20the%20beginning%20of%20your%20request%20handler%20(or%20whenever%20you%20want%20to%20reset%20the%20context%2Dlocal%20context).).

Reply:
Same as above, not needed here, removed now.